### PR TITLE
[1.22.0] release notes

### PR DESCRIPTION
**Pull request by salahalsaleh**

### CODE REVIEW

File: content/release-notes/1.22.0.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2020-10-30
+linktitle: "1.22.0"
+title: "1.22.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<features>}}
+  * Added support for disaster recovery backups and restores. Restores can be created both on the same instance or on a different instance. Embedded clusters do not support this feature yet.
```
Comment (markpundsack) [inline content/release-notes/1.22.0.md:9]:
```suggestion
  * Added beta support for disaster recovery backups and restores of instance snapshots, which include the Admin Console and all installed applications. Backups can be restored to the original instance or to a different instance. Instance snapshots do not yet support embedded clusters.
```

File: content/release-notes/1.22.0.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+{{</features>}}
+
+{{<fixes>}}
+* Fixed a bug where base64 encoded application icons do not render in airgapped installations.
+* Fixed a total snapshot capture time.
```
Comment (markpundsack) [inline content/release-notes/1.22.0.md:14]:
```suggestion
* Fixed reporting of total snapshot capture time.
```

File: content/release-notes/1.22.0.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+* Fixed remaining seconds when restoring volumes from application backups.
```
Comment (markpundsack) [inline content/release-notes/1.22.0.md:15]:
```suggestion
* Fixed reporting of remaining seconds when restoring volumes from application backups.
```

File: content/release-notes/1.22.0.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+* Fixed a bug in the KOTS Admin Console UI where a failure from "Check for updates" was not properly indicated.
```
Comment (markpundsack) [inline content/release-notes/1.22.0.md:16]: Hmm... this doesn't make much sense. What's the failure? Is this about better error reporting?

Comment (salahalsaleh) [reply content/release-notes/1.22.0.md:16]: i believe so yeah

**Pull request merged**